JetMETCorrections/Type1MET/interface/CorrectedMETProducerT.h

```diff
@@ -82,6 +84,13 @@ class CorrectedMETProducerT : public edm::stream::EDProducer<>
     delete algorithm_;
   }
     
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<std::string>("@module_label");
```

Comment:
It is not needed (done by the FW internally) and harms as @ is a non-standard character....